demisto_sdk/commands/content_graph/neo4j_service.py

```diff
@@ -122,7 +126,7 @@ def _neo4j_admin_command(name: str, command: str):
         try:
             docker_client.containers.get(f'neo4j-{name}').remove(force=True)
         except Exception as e:
-            logger.info(f'Could not remove neo4j container: {e}')
+            logger.debug(f'Could not remove neo4j container: {e}')
```

Comment:
```suggestion
            logger.info(f'Could not remove neo4j container: {e}')
```
Don't hide this error. You may not be able to start a new container as the port will be taken by the old container.

Reply:
If the container doesn't exist in the user env, we expect to get this "error" and this is ok.
This flow will remove the container if exists